http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -461,4 +544,124 @@ private void addScopes(String scopes, Set<String> allScopes) {
             allScopes.addAll(Arrays.asList(scopes.split("\\s+")));
         }
     }
+
+    private String convertToRequestParameter(URIBuilder redirectUriBuilder, String redirectUri, String state, List<NameValuePair> forwardedQueryParams) throws JoseException, IOException {
+        redirectUriBuilder.addParameter(SCOPE, OIDC_SCOPE);
+
+        JwtClaims jwtClaims = new JwtClaims();
+        jwtClaims.setIssuer(deployment.getIssuerUrl());
```

Comment:
From the spec, "the iss value SHOULD be the Client ID of the RP, unless it was signed by a different party than the RP". So here, we can use getResourceName.